src/index.html

```diff
@@ -6,17 +6,49 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
     />
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo" class="img" />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
```

Comment:
Keep your attributes correctly formatted.

```suggestion
          <li class="list_item">
            <a 
              href="#" 
              class="nav_link is-active"
            >
              Apple
            </a>
          </li>
```

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/8750a696-df99-4917-a559-ab6ef54fb041)

Reply:
Yes, I knew that only 2 attributes in a single line are allowed. But the Prettier reduces everything to one attribute in a line. That's why I changed my formatter's default settings. But okay. I'll fix it the way you wish